Tests/SentryTests/Integrations/SessionReplay/SentrySessionReplayTests.swift

```diff
@@ -164,6 +171,24 @@ class SentrySessionReplayTests: XCTestCase {
         assertFullSession(sut, expected: true)
     }
     
+    func testReplayScreenNames() throws {
+        let fixture = startFixture()
+        let sut = fixture.getSut(options: SentryReplayOptions(sessionSampleRate: 1, errorSampleRate: 1))
+        sut.start(rootView: fixture.rootView, fullSession: true)
+        
+        for i in 1...6 {
+            fixture.currentScreen = "Screen \(i)"
+            fixture.dateProvider.advance(by: 1)
+            Dynamic(sut).newFrame(nil)
+        }
+                
+        let replayEvent = try XCTUnwrap(fixture.lastReplayEvent)
+        
+        XCTAssertEqual(replayEvent.urls?[0], "Screen 1")
+        XCTAssertEqual(replayEvent.urls?[1], "Screen 2")
+        XCTAssertEqual(replayEvent.urls?[2], "Screen 3")
```

Comment:
```suggestion
        let urls = try XCTUnwrap(fixture.lastReplayEvent.urls)
        
        guard urls.count == 3 else {
        	XCTFail("Expected 3 screen names")
        	return
        }
        XCTAssertEqual(urls[0], "Screen 1")
        XCTAssertEqual(urls[1], "Screen 2")
        XCTAssertEqual(urls[2], "Screen 3")
```
need to guard against index out-of-bounds errors from crashing the test process